client/src/components/tab/TabComponent.tsx

```diff
@@ -1,82 +1,37 @@
 import * as React from 'react';
-import Tabs from '@mui/material/Tabs';
-import Tab from '@mui/material/Tab';
-import Box from '@mui/material/Box';
-import Paper from '@mui/material/Paper';
-import TabPanel, { a11yProps } from 'components/tab/subcomponents/TabPanel';
-import { useState, SyntheticEvent } from 'react';
+import { Paper } from '@mui/material';
+import TabRender from './subcomponents/TabRender';
```

Comment:
There seems to be a circular dependency between TabComponents and TabRender.

Reply:
Also as a convention, things in parent directories depend on sub-directories, except when they are in different packages, or in different parts of directory structure. For example, code in src/components/subcomponents/TabRender.tsx can depend on src/utility/sample.tsx but not on src/components/Components.tsx.

Reply:
@DaloonOfDoom took a look at this and fixed it